include/vinecopulib/vinecop/implementation/class.ipp

```diff
@@ -1087,59 +1082,110 @@ Vinecop::get_npars() const
 //! More generally, `Vinecop::rosenblatt()` returns the variables
 //! \f[ U_{M[d - j, j]}= F(V_{M[d - j, j]} | V_{M[d - j - 1, j - 1]}, \dots, V_{M[0, 0]}), \f]
 //! where \f$ M \f$ is the structure matrix. Similarly, `Vinecop::inverse_rosenblatt()`
+//!
+//! If some variables have atoms, Brockwell (10.1016/j.spl.2007.02.008) proposed a
+//! simple randomization scheme to ensure that output is still independent uniform
```

Comment:
I think that this should give a bit more details on the randomization scheme that is used. Maybe just the formula?